lib/pf/UnifiedApi/Controller/Config/Bases.pm

```diff
@@ -81,6 +81,14 @@ sub test_smtp {
     return $self->render(json => { message => 'Testing SMTP success' });
 }
 
+=head2 fields_to_mask
+
+fields_to_mask
+
+=cut
+
+sub fields_to_mask { qw(smtp_password pass galera_replication_password) }
```

Comment:
You should also go through the fields of the scan engines and the firewall sso entries

Reply:
By default, any field named password will get masked.
Will verify if the scan engines have other fields that need to be masked.

Reply:
They are good.